frontend/packages/topology/integration-tests/support/step-definitions/topology/list-view.ts

```diff
@@ -0,0 +1,52 @@
+import { When, Then } from 'cypress-cucumber-preprocessor/steps';
+import { navigateTo } from '@console/dev-console/integration-tests/support/pages/app';
+import { devNavigationMenu } from '@console/dev-console/integration-tests/support/constants/global';
+import { topologyPO } from '../../page-objects/topology-po';
+
+When('user clicks on List view button', () => {
+  navigateTo(devNavigationMenu.Topology);
+  if (cy.get(topologyPO.graph.emptyGraph)) {
+    cy.get(topologyPO.switcher).click();
+  } else {
+    cy.log('You are already on List View');
+  }
+});
+
+Then('user will see workloads are segregated by applications groupings', () => {
+  cy.get(topologyPO.graph.applicationGroupingTitle).should('be.visible');
+});
+
+When('user clicks the filter by resource on top', () => {
+  cy.get(topologyPO.graph.filterByResource.filterByResourceDropDown)
+    .should('contain', 'Filter by resource')
+    .click();
+});
+
+Then('user will see Deployment and Deployment-Config options with 1 associated with it', () => {
```

Comment:
`Deployment-Config` can be `DeploymentConfig` and be consistent

Reply:
Updated.